pypyr/cli.py

```diff
@@ -6,36 +6,63 @@
 import os
 import pypyr.pipelinerunner
 import pypyr.version
+import signal
+import sys
 
 
-def get_args():
+def get_args(args):
```

Comment:
Might be worth looking at the click library that cloco-cli uses.  Takes all of the hassle out of command line argument parsing.